walk-in-interview/src/main/java/com/google/job/data/JobsDatabase.java

```diff
@@ -233,4 +235,30 @@
             MoreExecutors.directExecutor()
         );
     }
+
+    /**
+     * Gets all the applicant's interested jobs given the params.
+     *
+     * @param pageSize The the number of jobs to be shown on the page.
+     * @param pageIndex The page number on which we are at.
+     * @return Future of the JobPage object.
+     */
+    public static Future<JobPage> fetchInterestedJobPage(int pageSize, int pageIndex) throws IOException {
+        CollectionReference applicantAccountsCollection = FireStoreUtils.getFireStore().collection(APPLICANT_ACCOUNTS_COLLECTION);
+
+        // TODO(issue/91): get userId from firebase session cookie
+        String applicantId = "";
```

Comment:
You should pass this as an argument instead. The jobs db doesn't care about session data, that's under purview of the servlet.